tests/apitests/api/test_helper.c

```diff
@@ -372,6 +373,12 @@ int add_friend_anyway(TestContext *context, const char *userid,
     CU_ASSERT_EQUAL_FATAL(rc, 2);
     CU_ASSERT_STRING_EQUAL_FATAL(buf[0], "fadd");
     CU_ASSERT_STRING_EQUAL_FATAL(buf[1], "succeeded");
+    if(strcmp(buf[0], "fadd") != 0) {
+        vlogE("======== fadd: %s", buf[0]);
+    }
+    if(strcmp(buf[1], "succeeded") != 0) {
+        vlogE("======== succeeded: %s", buf[1]);
+    }
```

Comment:
This block statements like to used for debug purpose. Is it necessary to keep them ?